exp-player/addon/components/exp-video-physics.js

```diff
@@ -0,0 +1,172 @@
+        }
+    },
+    actions: {
+        playNext: function() {
+            window.clearTimeout(this.get('timeoutID'));
+            if (this.get('doingIntro')) { // moving to test video
+                this.set('doingIntro', false);
+                $('#player-video')
+                    .attr('loop', true)
+                    .addClass('test')
+                    .removeClass('intro attn');
+            } else {
+                this.get('next')(); // moving to intro video
+                $('#player-video')
+                    .removeClass('attn test')
+                    .addClass('intro')
+                    .attr('loop', false);
+            }
+        },
+        startVideo: function () {
+            if (!this.get('doingIntro') && !this.get('doingAttn')) {
+                var emberObj = this;
+                var t = window.setTimeout(function(emb) {
+                    emb.send('playNext');
+                    }, emberObj.get('testLength')*1000, emberObj);
+                this.set('timeoutID', t);
+            }
+        },
+        pause: function () {
+            var movie = $('#player-video')[0];
+            var emberObj = this;
+            window.clearTimeout(this.get('timeoutID'));
+            if (!emberObj.get('doingAttn') || !this.checkFullscreen()) { // pausing one of the videos
+                emberObj.set('doingAttn', true);
+                $('#player-video')
+                    .addClass('attn')
+                    .removeClass('intro test')
+                    .attr('loop', true);
+            } else { // returning to the videos
+                 // if doing intro, just return, no change necessary.
+                 // but if we were on a test video...
+                if (!emberObj.get('doingIntro')) {
+                    // if it was already the alternate, just move on.
+                    if (emberObj.get('useAlternate')) {
+                        emberObj.send('next');
+                    } else {
+                    // if we still have the alternate to use, start at intro
+                        emberObj.set('useAlternate', true);
+                        emberObj.set('doingIntro', true);
+                    }
+                }
+
+                $('#player-video').removeClass('attn test');
+                $('#player-video').addClass('intro');
+                emberObj.set('doingAttn', false);
+                $('#player-video').attr('loop', false);
+            }
+        }
+    },
+
+    didRender() { // set up event handler for pausing
+        this._super(...arguments);
+        var emberObj = this;
+        $(document).on("keypress", function (e) {
+            if (emberObj.checkFullscreen()) {
+                if (e.which == 32) { // space
+                    emberObj.send('pause');
+                    // This isn't actually a function but somehow triggering the error works anyway so I'll take it for now...
+                    //emberObj.get('refresh')();
```

Comment:
So I believe the issue with double-clicking was caused by multiple event handlers being attached here. Basically `didRender` gets run every time the component re-renders, which in this case was happening when the video sources were being swapped out. Changing this to `init` limits the event binding only to the first render.

Proposed fix here: https://github.com/CenterForOpenScience/exp-addons/pull/73/files#diff-848af9bc2447ff0e1ce2bc717ba79035R173